trade-finance-manager-api/src/v1/api.js

```diff
@@ -250,7 +250,7 @@ const createFacilityAmendment = async (facilityId) => {
       const response = await axios({
         method: 'post',
         url: `${centralApiUrl}/v1/tfm/facilities/${facilityId}/amendments`,
-        headers: { 'Content-Type': 'application/json' },
+        headers: { 'Content-Type': 'application/json', 'x-api-key': API_KEY, },
```

Comment:
please use `headers.centralApi` and `headers.externalApi` for all the calls.

Reply:
can I just keep it as headers as both will be the same?